### ISSUE TITLE
TextModelPart is disposed!

### ISSUE THREAD

**Issue by VSCodeTriageBot** [microsoft/vscode#196672]
```javascript
Error: TextModelPart is disposed!
at f.g in src/vs/editor/common/model/textModelPart.ts:17:10
at f.getWordAtPosition in src/vs/editor/common/model/tokenizationTextModelPart.ts:208:8
at x.getWordAtPosition in src/vs/editor/common/model/textModel.ts:1964:42
at T.D in src/vs/editor/contrib/wordHighlighter/browser/wordHighlighter.ts:528:21
at T.F in src/vs/editor/contrib/wordHighlighter/browser/wordHighlighter.ts:612:22
at T.C in src/vs/editor/contrib/wordHighlighter/browser/wordHighlighter.ts:520:8
at h.value in src/vs/editor/contrib/wordHighlighter/browser/wordHighlighter.ts:292:9
at d.y in src/vs/base/common/event.ts:1140:13
at d.z in src/vs/base/common/event.ts:1151:9
at d.fire in src/vs/base/common/event.ts:1175:9
at h.value in src/vs/editor/browser/widget/codeEditorWidget.ts:1714:38
at d.y in src/vs/base/common/event.ts:1140:13
at d.fire in src/vs/base/common/event.ts:1171:9
at I.s in src/vs/editor/common/viewModelEventDispatcher.ts:64:18
at I.endEmitViewEvents in src/vs/editor/common/viewModelEventDispatcher.ts:109:8
at m.S in src/vs/editor/common/viewModel/viewModelImpl.ts:1092:26
at m.setCursorStates in src/vs/editor/common/viewModel/viewModelImpl.ts:987:15
at N.runCoreEditorCommand in src/vs/editor/browser/coreCommands.ts:392:41
at L.moveTo in src/vs/editor/browser/view/viewController.ts:224:33
at L.dispatchMouse in src/vs/editor/browser/view/viewController.ts:207:11
at i.G in src/vs/editor/browser/controller/mouseHandler.ts:610:24
at i.start in src/vs/editor/browser/controller/mouseHandler.ts:485:8
at v.G in src/vs/editor/browser/controller/mouseHandler.ts:327:29
at <anonymous> in src/vs/editor/browser/controller/mouseHandler.ts:130:83
at callback in src/vs/editor/browser/editorDom.ts:165:4
```
[Go to Errors Site](https://errors.code.visualstudio.com/card?ch=7dff07893ac160244db210e574eccc05f1cba2cb&bH=87ecb242-240a-6029-32b6-ee6cd88285df)

**Comment by bpasero** [comment]
This fills our insiders error telemetry many pages and looks like a top hitter.

**Comment by Yoyokrazy** [comment]
I'll keep an eye on this, but I think this was closed yesterday with #196405 being released. Error was last hit 16 hours ago, will keep an eye out.

**Comment by Yoyokrazy** [comment]
Still no hits, pretty confident in this being fixed. Will close, and reopen if I see any activity.
